### PR TITLE
[160015501] Add Economy tab under National Circumstances

### PR DESCRIPTION
**Pull request by aabdaab**
Add Economy tab along with its subsections:
- GDP
- GDP Growth
- Human Development Index

GDP:
image

GDP Growth:
image

HDI:
image

TODO:
When the data is ready, change selectors, update providers, check the axises configs.

### CODE REVIEW

**Comment by aabdaab** [comment]
Please don't hate me for the size of this PR...

File: app/javascript/app/pages/national-circumstances/economy/human-development-index/human-development-index-component.jsx [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+            slug="economy"
+            handleInfoClick={this.handleInfoClick}
+          />
+        </div>
+        <div className={styles.chart}>
+          <Chart
+            type="line"
+            lineType="linear"
+            dots={false}
+            customMessage="Emissions data not available"
```
Comment (Bluesmile82) [inline app/javascript/app/pages/national-circumstances/economy/human-development-index/human-development-index-component.jsx]: Please change this to Economy data

File: app/javascript/app/pages/national-circumstances/economy/gdp-growth/gdp-tooltip-chart/gdp-tooltip-chart-component.jsx [hunk 2]
```diff
@@ -0,0 +1,90 @@
...
+                    <div key={`${y.dataKey}`} className={styles.label}>
+                      <p className={styles.labelValue}>
+                        {this.renderValue(y)}
+                      </p>
+                    </div>
+)
+                  : null
+            )
+        }
+        {content && !content.payload && <div>No data fool</div>}
```
Comment (Bluesmile82) [inline app/javascript/app/pages/national-circumstances/economy/gdp-growth/gdp-tooltip-chart/gdp-tooltip-chart-component.jsx]: The human development index tooltip is not working. Also, could we change this message to 'No data'?

File: app/javascript/app/pages/national-circumstances/economy/gdp/gdp-component.jsx [hunk 3]
```diff
@@ -0,0 +1,104 @@
...
+      <Dropdown
+        theme={{ wrapper: styles.dropdown }}
+        options={metricOptions}
+        value={metricSelected}
+        onValueChange={this.handleMetricChange}
+        hideResetButton
+      />
+    );
+    const toolbar = (
+      <InfoDownloadToolbox
```
Comment (Bluesmile82) [inline app/javascript/app/pages/national-circumstances/economy/gdp/gdp-component.jsx]: Can we disable the download button until it has its functionality ready?

**Comment by Bluesmile82** [comment]
Isn't the GDP grown supposed to be a percentage. Maybe its just because its mocked data. But the labels in the axis might change.
image

**Comment by Bluesmile82** [comment]
Same in GDP, maybe because CO2E is the default value:
image

File: app/javascript/app/pages/national-circumstances/economy/gdp-growth/gdp-tooltip-chart/gdp-tooltip-chart-component.jsx (hunk 2 above)
Comment (aabdaab) [reply app/javascript/app/pages/national-circumstances/economy/gdp-growth/gdp-tooltip-chart/gdp-tooltip-chart-component.jsx]: how do you mean? 
screen shot 2018-09-05 at 11 41 20

File: app/javascript/app/pages/national-circumstances/economy/human-development-index/human-development-index-component.jsx (hunk 1 above)
Comment (aabdaab) [reply app/javascript/app/pages/national-circumstances/economy/human-development-index/human-development-index-component.jsx]: good catch. thanks!

File: app/javascript/app/pages/national-circumstances/economy/gdp/gdp-component.jsx (hunk 3 above)
Comment (aabdaab) [reply app/javascript/app/pages/national-circumstances/economy/gdp/gdp-component.jsx]: sure

**Comment by aabdaab** [comment]
the GDP is taken from the CW API, so you're right about that, it should be changed (here I did not use mock data)
although for the GDP Growth I did change the axis:
screen shot 2018-09-05 at 11 43 35

File: app/javascript/app/providers/gdp-growth-provider/dummy.json [hunk 4]
```diff
@@ -0,0 +1,70 @@
+{
+  "config": {
+    "axes": {
+      "xBottom": { "name": "Year", "unit": "date", "format": "YYYY" },
+      "yLeft": {
+        "name": "GDP Growth",
+        "unit": "",
```
Comment (Bluesmile82) [inline app/javascript/app/providers/gdp-growth-provider/dummy.json]: Maybe we should specify the percentage unit and prepare the table for this kind of data

Comment (aabdaab) [reply app/javascript/app/providers/gdp-growth-provider/dummy.json]: it's a good idea. what kind of a table do you mean?
I was also thinking about moving the `axis-ticks` files to maybe a common folder and have all of them grouped there with meaningful names, like `PercentageYAxis` etc.
Also for the tooltips.
WDYT?

**Comment by aabdaab** [comment]
Small fixes are already pushed but I still want to refactor the different axes and tooltips handling.

File: app/javascript/app/providers/gdp-growth-provider/dummy.json (hunk 4 above)
Comment (Bluesmile82) [reply app/javascript/app/providers/gdp-growth-provider/dummy.json]: I think that could work well. We can reuse them for sure. Sorry, for table I meant graph.

**Comment by Bluesmile82** [comment]
Great work! Thanks for the changes

**Pull request merged**